docs/howto/krustlet-on-eks.md

```diff
@@ -1,3 +1,173 @@
 # Running Krustlet on Amazon Elastic Kubernetes Service (EKS)
 
-TODO: https://github.com/deislabs/krustlet/issues/130
+Currently, [EKS does not support](https://github.com/aws/containers-roadmap/issues/741) running managed node groups with custom Amazon Machine Images (AMI).
+
+However, it does appear the feature might be coming soon.
+
+Until that time, we can use [eksctl](https://eksctl.io/) to create and manage a node group with a custom Krustlet-based AMI.
+
+# Prerequisites
```

Comment:
Can all of these section headers be `h2` to match the style of the rest of the docs?

Reply:
I'll change.